tests/integration/test_basic_examples.py

```diff
@@ -0,0 +1,46 @@
+import aiohttp
+import pytest
+
+from .actions import encoder, screen, temperature, network
+from simulator import MachineType, Thermistor, Printer
+
+pytestmark = pytest.mark.asyncio
+
+
+@pytest.fixture
+def wui_base_url(printer):
+    return f'http://localhost:{network.proxy_http_port_get(printer)}'
+
+
+@pytest.fixture
+async def wui_client(wui_base_url):
+    async with aiohttp.ClientSession(base_url=wui_base_url) as session:
+        yield session
+
+
+async def test_max_temp_error_on_bed(printer: Printer):
+    await screen.wait_for_text(printer, 'HOME')
+    await temperature.set(printer, Thermistor.BED, 230)
+    await screen.wait_for_text(printer, 'MAXTEMP ERROR')
+
+
+async def test_web_interface_is_accessible(printer: Printer,
+                                           wui_client: aiohttp.ClientSession):
+    await screen.wait_for_text(printer, 'HOME')
```

Comment:
That might not be enough :-(. The home screen might appear sooner than the whole DHCP handshake happens.

Well, maybe not on tests, hopefully.

Reply:
Yep, might not be. Currently the best (and only) channel „fw->tests“ we have here is the OCR of the text on the screen. So let's hope it is gonna be enough 🙂

Reply:
Seems like this is fine. I guess that the DHCP exchange is fast enough in the simulator, and in addition, the client will try retransmitting the ARP/Syn packet for a while before giving up.